### ISSUE TITLE
normantranscript.com

### ISSUE THREAD

**Issue by NanoMeow** [NanoMeow/QuickReports#745]
### Basic Information

**Test link:** `https://www.normantranscript.com/opinion/use-the-safest-internet-browser/article_4d584b34-753d-54d4-bab3-154a2b2ba756.html`
**Category:** other
**Reported from:** Taiwan
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:65.0) Gecko/20100101 Firefox/65.0
**Extension:** Nano Defender 15.0.0.106

### Additional Message

Anti-private browsing (Preventing user browsing in Incognito mode)

**Comment by mapx-** [comment]
fixed as annoyance

**Comment by jspenguin2017** [comment]
Google is planning to put a lid on this Incognito detection nonsense: https://www.bleepingcomputer.com/news/google/google-fixing-chrome-api-to-prevent-incognito-mode-detection/

Not sure what to do in the meantime, I'm inclined to say won't fix.

**Comment by DandelionSprout** [comment]
I recall this being fixed in *uBlock Filters - Annoyances* two days ago, if I'm not wrong: https://github.com/uBlockOrigin/uAssets/blob/master/filters/annoyances.txt#L1817

**Comment by jspenguin2017** [comment]
@DandelionSprout Yes, that's right. I would have closed this if it's in the uBlock privacy filter like other anti-incognito websites. But since it's in the annoyance filter, I have yet to decide what to do.

**Comment by jspenguin2017** [comment]
`t: other`: Anti-Incognito

**Comment by jspenguin2017** [comment]
I don't see any incognito detection anymore, however, there is some ads placeholders.
